server/models/requestSchema.js

```diff
@@ -2,11 +2,19 @@ var mongoose = require('mongoose')
 
 const Schema = mongoose.Schema;
 
+const timeAvailabilitySchema = new Schema({
+  id: {type: String, required: true},
+  day: {type: String, enum: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'], required: true },
+  time: {type: Number, required: true}
+})
+
 const requestSchema = new Schema(
   {
-    student: {type: Schema.Types.ObjectId, ref: 'Student'},
-    alumni: {type: Schema.Types.ObjectId, ref: 'Alumni'},
-    time: {type: String, required: true},
+    requester: {type: String, required: true},
+    requesterObj: {type: Object, required: false},
```

Comment:
This is never stored when a user makes a request, but allows for the full profile to be returned when we need it (see mongooseUtil)